### ISSUE TITLE
OpenCV `Vec4iVector` should be built on `IntPointer`

### ISSUE THREAD

**Issue by jpsacha** [bytedeco/javacpp-presets#717]
In C++ code `Vec4i` is defined as a vector of integers:
```c++
typedef Vec<int, 4> cv::Vec4i
```
In `org.bytedeco.opencv.presets.opencv_imgproc` (line 53) it is defined based on `FloatPointer`:
```java
new Info(..., "cv::Vec4i", ...).cast().pointerTypes("FloatPointer"))
```
it should be rather:
```java
new Info("cv::Vec4i").cast().pointerTypes("IntPointer"))
```
A problem with current implementation can be seen when using `HoughLinesP`. It will produce lines' coordinates as Vec4i. Those coordinates of lines cannot be correctly transferred from the C++ side.

**Comment by saudet** [comment]
Yes, that's an error that slipped there. In any case, to work around that with the current release, we can easily cast to `IntPointer` to access it properly.

**Comment by jpsacha** [comment]
Not clear how workaround with casting to `IntPointer` should be done. 

Here is some sample code that is using `HoughLinesP` to get line fragments(`lines`):
```java
Vec4iVector lines = new Vec4iVector()
HoughLinesP(binary, lines, deltaRho, deltaTheta, minVotes, minLength, minGap)
FloatPointer l = lines.get(i)
float x1 = l.get(0)
```
Due to the bug, the value of `x1` is not extracted correctly. Where the casting to `IntPointer` should happen?

**Comment by saudet** [comment]
Something like this:
```
FloatPointer l = new IntPointer(lines.get(i));
int x1 = l.get(0);
```

**Comment by jpsacha** [comment]
The workaround works. I added it to the LineFinder example.

**Comment by saudet** [comment]
Fix included in version 1.5.1. Thanks for reporting!
